src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -668,6 +668,7 @@ protected TypeSymbolWithAnnotations BindEventType(Binder binder, TypeSyntax type
         internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions, DiagnosticBag diagnostics)
         {
             var location = this.Locations[0];
+            bool includeNullability = DeclaringCompilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes);
```

Comment:
>bool includeNullability = DeclaringCompilation.IsFeatureEnabled(MessageID.IDS_FeatureNullableReferenceTypes); [](start = 12, length = 109)

It looks like the local is not used.

Reply:
Missed that when resolving the conflicts with Rikki PR

---
In reply to: [255382462](https://github.com/dotnet/roslyn/pull/33261#discussion_r255382462) [](ancestors = 255382462)